style.css

```diff
@@ -470,3 +491,293 @@ main {
   margin: 0 auto;
   margin-top: 24px;
 }
+
+@media screen and (min-width: 786px) {
+  * {
+    margin: 0;
+    padding: 0;
+    text-decoration: none;
+    box-sizing: border-box;
+  }
+
+  body {
+    font-family: "Poppins", sans-serif;
+    font-size: 16px;
+    overflow-x: hidden;
+    width: 100%;
+  }
+
+  .general-container {
+    height: 100%;
+    background: #f4f5f7;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .central-wrapper {
+    width: 100%;
+    background-color: #fff;
+    border-bottom-left-radius: 114px;
+  }
+
+  .header-class {
+    width: 80vw;
+    height: 84px;
+  }
+
+  .desktop-nav {
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+    flex-direction: row;
+    list-style: none;
+    width: 40vw;
+    right: 20px;
+  }
+
+  .tech-toolss li {
+    width: 72px;
+    background-color: #f7f7f9;
+    margin-left: 5px;
+  }
+
+  .desktop-nav li {
+    margin-right: 2em;
+  }
+
+  .desktop-nav li:last-child {
+    margin-right: 0;
+  }
+
+  a:hover {
+    background-color: grey;
+    cursor: pointer;
+  }
+
+  .desktop-nav li a:hover {
+    background-color: #091e42;
+
+    /* color:#091e42 */
+  }
+
+  .header-typo {
+    font-weight: 500;
```

Comment:
I apologize if no reviewer mentioned this previously, but none of your `font-weights` are working properly. The reason for that is that you need to download all of the different `font-weights` directly from Google Fonts or wherever you are getting your fonts from.

In order to do so, you need to click on the `Select this style` button for all of the weights you'd like to add and then import the font the way you did it before.

<img width="1049" alt="Screen Shot 2022-06-02 at 19 01 27" src="https://user-images.githubusercontent.com/82426682/171759537-4b23ee8b-e8a4-4a24-ae91-88cd0b3eda5a.png">
 
Please leave a comment if my instructions are not clear.